gpt_index/readers/file/mbox_parser.py

```diff
@@ -0,0 +1,97 @@
+"""Mbox parser.
+
+Contains simple parser for mbox files.
+
+"""
+from pathlib import Path
+from typing import Any, Dict, List
+
+from gpt_index.readers.file.base_parser import BaseParser
+
+
+class MboxParser(BaseParser):
+    """Mbox parser.
+
+    Extract messages from mailbox files.
+    Returns string including date, subject, sender, receiver and
+    content for each message.
+
+    """
+
+    def __init__(
+        self,
+        *args: Any,
+        max_count: int = 0,
```

Comment:
could probably be None by default (and if not None we validate that max_count > 0)